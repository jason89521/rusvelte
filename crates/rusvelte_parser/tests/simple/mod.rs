use crate::test_success;

test_success!();
