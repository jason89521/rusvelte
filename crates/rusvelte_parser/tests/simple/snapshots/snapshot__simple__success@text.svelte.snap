---
source: crates/rusvelte_parser/tests/simple/mod.rs
expression: root
input_file: crates/rusvelte_parser/tests/simple/inputs/text.svelte
snapshot_kind: text
---
{
  "type": "Root",
  "css": null,
  "start": 0,
  "end": 8,
  "fragment": {
    "type": "Fragment",
    "nodes": [
      {
        "type": "Text",
        "start": 0,
        "end": 8,
        "raw": "text&lt;",
        "data": "text<"
      }
    ]
  },
  "module": null,
  "instance": null,
  "options": null
}
